SCClassLibrary/Common/Streams/FilterPatterns.sc

```diff
@@ -445,11 +445,11 @@ Pfindur : FilterPattern {
 }
 
 Psync : FilterPattern {
-	var <>quant, <>maxdur, <>tolerance;
-	*new { arg pattern, quant, maxdur, tolerance = 0.001;
-		^super.new(pattern).quant_(quant).maxdur_(maxdur).tolerance_(tolerance)
+	var <>quant, <>maxdur, <>tolerance, <>mindur;
+	*new { arg pattern, quant, maxdur, tolerance = 0.001, mindur = 0;
+		^super.new(pattern).quant_(quant).maxdur_(maxdur).tolerance_(tolerance).mindur_(mindur)
 	}
-	storeArgs { ^[pattern,quant,maxdur,tolerance] }
+	storeArgs { ^[pattern,quant,maxdur,tolerance,mindur] }
```

Comment:
Let's add spaces after commas (since this is already touching that line).

Sorry, I had tried to include this in the other review but it got lost.